components/org.wso2.carbon.siddhi.editor.core/src/main/resources/web/js/sample-view/sample-view.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+define(['require', 'jquery', 'lodash','log'], function (require, $,  _,log) {
+
+    var ServiceView = function(config){
+        this._config = config;
+    };
+
+    ServiceView.prototype.constructor = ServiceView;
+
+    ServiceView.prototype.render = function () {
+        var config = this._config;
+        var errMsg;
+        if (!_.has(config, 'parentContainer')) {
+            errMsg = 'unable to find configuration for parentContainer';
+            log.error(errMsg);
+            throw errMsg;
+        }
+        // get parent container which is innerSamples div
+        var parentContainer = _.get(config, 'parentContainer');
+
```

Comment:
@kts-desilva could you please remove unnecessary spaces within methods

Reply:
Sure. Fixed in f228ff475e8dd19e1ef997ba24528c91a61ff51e